content/docs/intro/using_helm.md

```diff
@@ -36,49 +36,50 @@ installation. And to find new charts, you can search Helm chart _repositories_.
 
 ## 'helm search': Finding Charts
 
-When you first install Helm, it is preconfigured to talk to the official
-Kubernetes charts repository. This repository contains a number of carefully
-curated and maintained charts. This chart repository is named `stable` by
-default.
+Helm comes with a powerful  search command. It can be used to search two different
```

Comment:
Extra whitespace character between "powerful" and "search"

Reply:
```suggestion
Helm comes with a powerful search command. It can be used to search two different
```